src/main/python/docs/source/lineage.rst

```diff
@@ -0,0 +1,99 @@
+.. ------------------------------------------------------------------------------
+..  Copyright 2020 Graz University of Technology
+..
+..  Licensed under the Apache License, Version 2.0 (the "License");
+..  you may not use this file except in compliance with the License.
+..  You may obtain a copy of the License at
+..
+..    http://www.apache.org/licenses/LICENSE-2.0
+..
+..  Unless required by applicable law or agreed to in writing, software
+..  distributed under the License is distributed on an "AS IS" BASIS,
+..  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+..  See the License for the specific language governing permissions and
+..  limitations under the License.
+.. ------------------------------------------------------------------------------
+
+
+Lineage Trace API
+=================
+
+Lineage tracing captures every instruction during runtime and store those in a DAG. 
+An user can retreive the string representation of a lineage trace by enabling 
+this feature. Each line of a lineage trace, which is called *lineage item*, 
+represents a single operation. 
+A *lineage item* is composed of an *id*, which is uniquely identifiable 
+through an execution, an *opcode* for identifying the physical operation, 
```

Comment:
The *id* is uniquely identifiable through an execution? Do you mean, that the *id* uniquely identifies the lineage item throughout a single execution?